tests/SearchTest.java

```diff
@@ -0,0 +1,8 @@
+import static org.junit.Assert.*;
+
+
+
+public class SearchTest {
+
```

Comment:
Looking forward to seeing the test here ;)

Reply:
Added first test